### PR TITLE
Inline an implementation of the tokenList function

### LINKED ISSUE #27: Upgrade expressions that use tokenList

### PR DESCRIPTION
**Pull request by rictic**
... when we detect that it's being used in 0.5 code.

Fixes #27

### CODE REVIEW

File: lib/upgrade_js.js [hunk 1]
```diff
@@ -218,6 +219,25 @@ class Script {
       declarationNames.add(getDeclName(decl));
       declaration.properties.push(decl);
     });
+
+    if (elementInfo.polyfillTokenList) {
+      const ast = espree.parse(`
+        (function (obj) {
+          var pieces = [];
+          for (key in obj) {
```
Comment (azakus) [inline lib/upgrade_js.js]: Did tokenList run on prototypical properties as well? If not, we should use `Object.keys`.

Comment (azakus) [inline lib/upgrade_js.js]: Nvm, it did.

**Comment by azakus** [comment]
LGTM

File: lib/upgrade_js.js (hunk 1 above)
Comment (rictic) [inline lib/upgrade_js.js]: Looks like it did run on prototypical properties too.

**Pull request merged**